lepton3.go

```diff
@@ -87,34 +88,31 @@ func (d *Lepton3) SetLogFunc(log func(string)) {
 	d.log = log
 }
 
+// SetRadiometry enables or disables radiometry mode. If enabled, the
+// camera will attempt to automatically compensate for ambient
+// temperature changes.
 func (d *Lepton3) SetRadiometry(enable bool) error {
-	d.log("opening I2C bus")
-	i2cBus, err := i2creg.Open("")
+	cciDev, err := openCCI()
 	if err != nil {
 		return err
 	}
-	defer i2cBus.Close()
-
-	d.log("opening CCI")
-	cciDev, err := cci.New(i2cBus)
-	if err != nil {
-		return fmt.Errorf("cci.New: %v", err)
-	}
+	defer cciDev.Close()
 
-	d.log(fmt.Sprintf("setting radiometry to %v", enable))
 	if err := cciDev.SetRadiometry(enable); err != nil {
 		return fmt.Errorf("SetRadiometry: %v", err)
 	}
+	return nil
+}
 
-	d.log("checking radiometry setting is correct")
-	value, err := cciDev.GetRadiometry()
+// RunFFC forces the camera to run a Flat Field Correction
+// recalibration.
+func (d *Lepton3) RunFFC() error {
+	cciDev, err := openCCI()
```

Comment:
Little bit confusing that you use cciDev for both cci.Dev and closingCCIDev.  Maybe cCCIDev would be better?

Reply:
Well it's only used for closingCCIDev now...